src/modules/borrow-history/dto/borrow-history.dto.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { BorrowHistory } from '../domain/borrow-history';
+export class BorrowHistoryDto {
+  @ApiProperty()
+  readonly id: number;
+
+  @ApiProperty()
+  readonly userId: number;
+
+  @ApiProperty()
+  readonly bookId: number;
+
+  @ApiProperty()
+  readonly borrowDate: Date;
+
+  @ApiProperty({ required: false })
```

Comment:
DTO dùng cho việc response thì không cần mấy field này


```suggestion
  @ApiProperty()
```